### PR TITLE
SEO-175653-WPF-Links-With-Redirects

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

Worked on the links with redirects.

Regards,
Beryl.

### CODE REVIEW

File: wpf/DataGrid/UI-Automation.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Coded UI Test Builder generates code from recorded session and custom class is i
 
 ### Requirements and Configuration
 
-Coded UI provides support only in Visual Studio Ultimate and Visual Studio Premium. For more information about the platforms and configurations that are supported by Coded UI tests, refer this [link](https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2015/test/supported-configurations-and-platforms-for-coded-ui-tests-and-action-recordings?view=vs-2015&redirectedfrom=MSDN).
+Coded UI provides support only in Visual Studio Ultimate and Visual Studio Premium. For more information about the platforms and configurations that are supported by Coded UI tests, refer this [link](https://learn.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2015/test/supported-configurations-and-platforms-for-coded-ui-tests-and-action-recordings?view=vs-2015&redirectedfrom=MSDN).
 
 To test SfDataGrid with CUITs, build the Extension Project and place it in the mentioned location. You can get the Extension Project of SfDataGrid from [this](http://www.syncfusion.com/downloads/support/directtrac/general/ze/Src-229533545.zip) location.
```
[the remark this answers is not in the document]
Comment (berylouma) [reply wpf/DataGrid/UI-Automation.md]: Updated

File: wpf/DataGrid/Merge-Cells.md [hunk 2]
```diff
@@ -382,7 +382,7 @@ dataGrid.GetVisualContainer().InvalidateMeasureInfo();
 
 You can trigger [SfDataGrid.QueryCoveredRange](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfDataGrid.html) event for particular cell by removing its range from [SfDataGrid.CoveredCells](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfDataGrid.html#Syncfusion_UI_Xaml_Grid_SfDataGrid_CoveredCells) and invalidating the [VisualContainer](http://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.VisualContainer.html) by calling [InvalidateMeasureInfo](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.VisualContainer.html#Syncfusion_UI_Xaml_Grid_VisualContainer_InvalidateMeasureInfo) method. You can get range for particular cell in CoveredCells by passing row and column index to[SfDataGrid.CoveredCells.GetCoveredCellInfo](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.CoveredCellInfoCollection.html#Syncfusion_UI_Xaml_Grid_CoveredCellInfoCollection_GetCoveredCellInfo_System_Int32_System_Int32_) method.
```
Comment (Yvone-Atieno) [inline wpf/DataGrid/Merge-Cells.md:383]: Cr link not changed
image

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: wpf/DataGrid/Merge-Cells.md (hunk 2 above)
Comment (berylouma) [reply wpf/DataGrid/Merge-Cells.md]: Changed

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Fine.

**Pull request merged**